File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ActionType.java
package ru.iqchannels.sdk.schema;

import com.google.gson.annotations.SerializedName;

public enum ActionType {
    @SerializedName("Postback") POSTBACK,
    @SerializedName("Open URL") OPEN_URL,
    @SerializedName("Say something") SAY_SOMETHING
}

File: iqchannels-sdk/src/main/java/ru/iqchannels/sdk/schema/ActionType.kt
package ru.iqchannels.sdk.schema

import com.google.gson.annotations.SerializedName

enum class ActionType {
	@SerializedName("Postback")
	POSTBACK,
	@SerializedName("Open URL")
	OPEN_URL,
	@SerializedName("Say something")
	SAY_SOMETHING
}
